### ISSUE TITLE
Updates to XMLPipeDB website

### ISSUE THREAD

**Issue by kdahlquist** [lmu-bioinformatics/xmlpipedb#30]
@dondi, has the XMLPipeDB website been jekyll-ized?

I want to try and make some updates to the site, but am not sure how to go about it.

**Comment by dondi** [comment]
No, unfortunately it’s still on the older `maven`-based site-building system. The source files are all there under _site_ and should be mostly recognizable HTML, but I would still need to regenerate the site files manually.

**Comment by kdahlquist** [comment]
@NAnguiano is going to make updates to the Publications page based on items from @kdahlquist's CV.

**Comment by kdahlquist** [comment]
As per #47, this is no longer assigned to @NAnguiano.

**Comment by kdahlquist** [comment]
As per #45, I found some broken links and a few minor things that need to be fixed on the website.  I'll plug away at this as I have time.
